src/store/reducers/createButtonReducer.js

```diff
@@ -0,0 +1,14 @@
+import { TOGGlE_CREATE_BUTTON_VISIBILITY } from "store/constants";
+
+const initialState = false;
```

Comment:
for just toggling i don't think using redux is necessary. You can use state for this operation